### ISSUE TITLE
Warning about msgpack not installed confuses node operators

### ISSUE THREAD

**Issue by hoh** [aleph-im/pyaleph#198]
When launching pyaleph, especially for operations such as key generation, node operators get confused by the following warnings:

```
ujson module not found, using json
msgpack not installed, MsgPackSerializer unavailable
```

These appear to come from a dependency of PyAleph and not PyAleph itself.

We have three possibilities to hide these:
- Fix it upstream
- Change the log level for the dependencies displaying the warnings
- Installing `ujson` and `msgpack` in PyAleph.

Note that `ujson` would probably increase the performance of the JSON packing and unpacking.
msgpack is only used by [aleph-vm](https://github.com/aleph-im/aleph-vm), not by the CCN.

**Comment by odesenfans** [comment]
I'll have a look.

**Comment by odesenfans** [comment]
Ok so both warnings come from `aiocache`. The reasonable way to fix these IMO is to install the missing packages.

**Comment by hoh** [comment]
OK, let's do that !

**Comment by odesenfans** [comment]
Resolved by #200.
